stable/velero/values.yaml

```diff
@@ -0,0 +1,95 @@
+image:
+  repository: gcr.io/heptio-images/velero
+  tag: v0.11.0
+  pullPolicy: IfNotPresent
+
+# Only kube2iam/kiam: change the AWS_ACCOUNT_ID and VELERO_ROLE_NAME
+podAnnotations: {}
+#  iam.amazonaws.com/role: arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME>
+#  prometheus.io/scrape: "true"
+#  prometheus.io/port: "8085"
+#  prometheus.io/path: "/metrics"
+
+rbac:
+  create: true
+
+resources: {}
+
+# this is the k8s spec block for initContainers:
+initContainers: []
+  # - name:
+  #   image:
+  #   volumeMounts:
+  #     - name: plugins
+  #       mountPath: /target
+
+
+serviceAccount:
+  server:
+    create: true
+    name:
+
+tolerations: []
+
+nodeSelector: {}
+
+## Parameters for the ' default' Config resource
+## See https://heptio.github.io/velero/v0.9.0/config-definition
+configuration:
+  provider:
+
+  volumeSnapshotLocation: {}
+  #  name:
+  #  config:
+  #    region:
+  #    apiTimeout:
+
+  backupStorageLocation:
+    name:
+    bucket:
+    config: {}
+    #  region:
+    #  prefix:
```

Comment:
The `prefix` parameter should be below backupStorageLocation (not below config). Check https://github.com/helm/charts/pull/11430 which also fixes this

Reply:
Done